index.yaml

```diff
@@ -217,6 +217,11 @@ indexes:
   - name: email_hash
   - name: sent_datetime
 
+- kind: SkillOpportunityModel
+  properties:
+  - name: deleted
+  - name: created_on
```

Comment:
Is this an expected (automated) change?

I don't see any changes in the gae_models.py file in this PR. Maybe you can undo this change and restart the server and if required this will again regenerate? (@seanlip am I right on this? Asking you as I'm not sure how the automated generation works!)

Reply:
Hi @DubeySandeep -- yes, you are correct, and sorry for missing this! And @sagangwee, @DubeySandeep is right about the regeneration steps -- the only instruction I would add is to do something that triggers use of the SkillOpportunityModel from the frontend if possible, and if the index.yaml file doesn't change, then this change is not needed.